docs/nima/fault-tolerance.adoc

```diff
@@ -26,5 +26,280 @@
 
 include::{rootdir}/includes/nima.adoc[]
 
+== Contents
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<API, API>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
+
+== Overview
+
+Helidon Nima Fault Tolerance support is inspired by
```

Comment:
We may need to use the correct formatting for Nima (N&iacute;ma) but we are still looking into some display issues.